features/provider/onboarding/wizard.feature

```diff
@@ -14,3 +14,18 @@ Feature: Onboarding Wizard
      And sends the test request
      And goes to what's next
     Then goes to API page
+
+  @emails
+  Scenario: Provider goes through the APIAP wizard
+    Given a provider signs up and activates his account
+    When user starts the onboarding wizard
+     And goes to Add a Product page
+     And adds the echo Product
+     And goes to Add a Backend page
```

Comment:
@thomasmaas
Do you know why this step isn't working? Is it because it's `And` and not `Then`?

Reply:
No, And and Then have no consequences for the test.